File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.cc
#include <iostream>
#include <cstdlib>
#include<iomanip>
using namespace std;

int main() {

	int num[1000], c, n;
	
	cin >> c;

	for (int i = 0; i < c; i++) {

		float sum = 0;
		float avr = 0;
		float count = 0;
		float per = 0;

		cin >> n;

		for (int i = 0; i < n; i++) {

			cin >> num[i];
			sum += num[i];

		}

		avr = sum / n;

		for(int i = 0; i < n; i++) {
			
			if(avr < num[i])
				count ++;
		}

		per = (count / n) * 100;

		cout << fixed << setprecision(3) << per << "%" << endl;
	}

	

	return 0;

}

File: 백준/Bronze/4344. 평균은 넘겠지/평균은 넘겠지.java
//import
import java.util.Scanner;
 
public class Main {
	public static void main(String[] args) {
	//scanner 선언
	Scanner scanner = new Scanner(System.in);
        
        //테스트 수 입력 받기
        int testcase = scanner.nextInt();
        
        //테스트 배열 선언
		int[] array;
        
        //각 테스트 계산
        for (int i = 0; i < testcase; i++) {
            //평균 계산을 위한 합
            double sum = 0.0;
            //확률 
            double per = 0.0;
            
            //각 테스트 결과 갯수
            int test = scanner.nextInt();
            
            array = new int[test];
            
            for (int j = 0; j < test; j++) {
                array[j] = scanner.nextInt();
                //각 테스트 결과 모두 더하기
                sum += array[j];
            }
            
            //평균
            double avr = sum / test;
            
            //평균 넘는 확률 구하기
            for (int k = 0; k < test; k++) {
                if (array[k] > avr)
                    per++;
            }
            //결과 출력 
            System.out.printf("%.3f%%\n",(per / test) * 100);
        } 
        
        //scanner 닫기
	scanner.close();
	}
}
